### PR TITLE
add MX_Spanish translation for README.md (close #3188)

### LINKED ISSUE #3188: Add spanish translation readme
Add spanish translation of readme.
I want to correct them and work on this issue as of Hackoberfest.

### PR DESCRIPTION
**Pull request by ferdox2**
### Description
This change adds the MX_Spanish translation to the README.md file.

### Affected components

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System
- [ ] Tests
- [ ] Other (list it)

### Related Issues

#3188

#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [x] No
- [ ] Yes
  - [ ] Updated docs with SQL for downgrading the catalog 

#### Metadata

Does this PR add a new Metadata feature?
- [x] No
- [ ] Yes
  - Does `run_sql` auto manages the new metadata through schema diffing?
    - [ ] Yes
    - [ ] Not required
  - Does `run_sql` auto manages the definitions of metadata on renaming?
    - [ ] Yes
    - [ ] Not required
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?
    - [ ] Yes
    - [ ] Not required

#### GraphQL
- [x] No new GraphQL schema is generated
- [ ] New GraphQL schema is being generated:
   - [ ] New types and typenames are correlated
   
   

#### Breaking changes

- [x] No Breaking changes
- [ ] There are breaking changes:

  1. Metadata API

     Existing `query` types:
     - [ ] Modify `args` payload which is not backward compatible
     - [ ] Behavioural change of the API
     - [ ] Change in response `JSON` schema
     - [ ] Change in error code
     

  2. GraphQL API

     Schema Generation:
     
     
     - [ ] Change in any `NamedType`
     - [ ] Change in table field names
     

     Schema Resolve:-
     
     - [ ] Change in treatment of `null` value for any input fields 
     

  3. Logging

     - [ ] Log `JSON` schema has changed
     - [ ] Log `type` names have changed

### CODE REVIEW

**Comment by ferdox2** [comment]
I was checking that there are others PR with this topic, but I checked them and I encountered some mistakes. In this PR I tried to avoid spelling mistakes. I hope you could consider my PR. Thanks anyway.

File: translations/README.mx_spanish.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+* **Interfaz de administración y Migraciones**: Interfaz de administración y esquema de migraciones inspiradas en Rails.
+* **Postgres** ❤️: Soporta los tipos de Postgres (PostGIS/geo-location, etc.), convierte las vistas a *gráficas*, ejecuta funciones almacenadas o procedimientos con mutaciones.
+
+Leer más en [hasura.io](https://hasura.io) y en [docs](https://docs.hasura.io).
+
+## Tabla de contenidos
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Tabla de contenidos**
+
+- [Inicio rápido:](#quickstart)
```
[the remark this answers is not in the document]
Comment (ferdox2) [reply translations/README.mx_spanish.md]: done

File: translations/README.mx_spanish.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+El motor de Hasura GraphQL es un potente servidor de GraphQL que provee de **APIs instantáneas y en tiempo real a través de Postgres**, con [**lanzadores de webhook**](event-triggers.md) basados en los eventos de la base de datos, y [**esquemas remotos**](remote-schemas.md) para la lógica de negocio.
+
+Hasura te ayuda a crear aplicaciones GraphQL respaldadas por Postgres o de manera gradual permite migrar a aplicaciones GraphQL desde aplicaciones que usan Postgres. 
+
+Leer más en [hasura.io](https://hasura.io) y en [docs](https://docs.hasura.io).
+
+------------------
+
+![Hasura GraphQL Engine Demo](assets/demo.gif)
```
[the remark this answers is not in the document]
Comment (ferdox2) [reply translations/README.mx_spanish.md]: fixed

File: translations/README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md)                   ￨ [🇬🇷 Ελληνικά](README.greek.md)  |
+| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md)                   ￨ [🇬🇷 Ελληνικά](README.greek.md)  | [:mx: Spanish](README.mx_spanish.md) |
```
[the remark this answers is not in the document]
Comment (ferdox2) [reply translations/README.md]: done

**Comment by ferdox2** [comment]
I did all the changes you request me and fix others that I noticed when I was reviewing.

File: README.md [hunk 4]
```diff
@@ -213,5 +213,6 @@ This readme is available in the following translations:
 - [Japanese :jp:](translations/README.japanese.md) (:pray: [@moksahero](https://github.com/moksahero))
 - [French :fr:](translations/README.french.md) (:pray: [@l0ck3](https://github.com/l0ck3))
 - [Greek 🇬🇷](translations/README.greek.md) (:pray: [@MIP2000](https://github.com/MIP2000))
+- [Spanish :mx:](/translations/README.mx_spanish.md)(:pray: [@ferdox2](https://github.com/ferdox2))
```
Comment (marionschleifer) [inline README.md]: The flag is not showing here.

File: translations/README.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md)                   ￨ [🇬🇷 Ελληνικά](README.greek.md)  |
+| Readme          | [:uk: English](../README.md) ￨ [:fr: Français](README.french.md) ￨ [:jp: 日本語](README.japanese.md) ￨ [🇬🇷 Ελληνικά](README.greek.md) ￨ [:mx: Spanish](README.mx_spanish.md) |
```
Comment (marionschleifer) [inline translations/README.md]: The Spanish flag is also not showing here.

File: translations/README.mx_spanish.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+Agrega la lógica de negocios de forma asíncrona, la cual es ejecutada en base a los eventos de la base de datos.
+Ideal para notificaciones, canales de datos de Postgres o procesamiento asíncrono - [leer más](../event-triggers.md).
+
+### Datos derivados o transformación de datos
+
+Transforma datos en Postgres o ejecuta la lógica de negocio en ellos para derivar otro conjunto de datos que puedan ser consultados usando el motor de GraphQL - [leer más](https://docs.hasura.io/1.0/graphql/manual/queries/derived-data.html).
+
+## Demos
+
+Revisa todos los ejemplos de aplicaciones en la carpeta de [community/examples](../community/examples).
```
Comment (marionschleifer) [inline translations/README.mx_spanish.md]: This link is broken.

File: translations/README.mx_spanish.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+    - [Otros métodos de despliegue](#otros-métodos-de-despliegue)
+- [Arquitectura](#arquitectura)
+- [Herramientas client-side](#herramientas-client-side)
+- [Agregar lógica de negocio](#agregar-lógica-de-negocio)
+    - [Esquemas remotos](#esquemas-remotos)
+    - [Ejecutar webhooks con eventos de la base de datos](#ejecutar-webhooks-con-eventos-de-la-base-de-datos)
+- [Demos](#demos)
+    - [Aplicaciones en tiempo real](#aplicaciones-en-tiempo-real)
+    - [Videos](#videos)
+- [Soporte y solución de problemas](#support--troubleshooting)
```
Comment (marionschleifer) [inline translations/README.mx_spanish.md]: This link is not working.

**Comment by marionschleifer** [comment]
@ferdox2 any updates on implementing these changes? 🙂

**Comment by ferdox2** [comment]
> @ferdox2 any updates on implementing these changes? 🙂

oh sorry, I'm fixing the issues along the day

File: README.md (hunk 4 above)
Comment (ferdox2) [reply README.md]: done

File: translations/README.md (hunk 5 above)
Comment (ferdox2) [reply translations/README.md]: done

File: translations/README.mx_spanish.md (hunk 6 above)
Comment (ferdox2) [reply translations/README.mx_spanish.md]: done

File: translations/README.mx_spanish.md (hunk 7 above)
Comment (ferdox2) [reply translations/README.mx_spanish.md]: done

**Comment by ferdox2** [comment]
@marionschleifer I fixed the issues, sorry for the late

File: translations/README.mx_spanish.md [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+## Contribuir
+
+Revisa nuestra [guía de contribución](../CONTRIBUTING.md) para más detalles.
+
+## Archivos de marca
+
+Los archivos de marca de Hasura (logotipos, la mascota de Hasura, la insignia: "powered by", etc.) pueden ser encontradas en la carpeta [assets/brand](../assets/brand). Siéntete libre de usarlos en tu
+aplicación, sitio web, etc. Estaremos encantados si agregas la insignia "Powered by Hasura" a tus aplicaciones creadas con Hasura. ❤️
+
+<div style="display: flex;">
```
Comment (marionschleifer) [inline translations/README.mx_spanish.md]: The Hasura logo doesn't show here:

image

File: translations/README.mx_spanish.md [hunk 9]
```diff
@@ -0,0 +1,210 @@
...
+
+Todos **los contenidos** (excepto aquellos incluidos en los directorios [`server`](../server), [`cli`](../cli) y
+[`console`](../console)) están disponibles bajo la [Licencia MIT](../LICENSE-community).
+Esto incluye todo en los directorios [`docs`](../docs) y [`community`](../community).
+
+## Traducciones
+
+Este archivo está disponible en los siguientes idiomas:
+
+- [Japonés :jp:](../translations/README.japanese.md) (:pray: [@moksahero](https://github.com/moksahero))
```
Comment (marionschleifer) [inline translations/README.mx_spanish.md]: There are more translations here by now. Could you add them please?

**Comment by marionschleifer** [comment]
@Skao93 would you mind going through the translation and checking the language? 🙂

File: translations/README.mx_spanish.md (hunk 9 above)
Comment (ferdox2) [reply translations/README.mx_spanish.md]: done

File: translations/README.mx_spanish.md (hunk 8 above)
Comment (ferdox2) [reply translations/README.mx_spanish.md]: done

**Pull request merged**